superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -197,7 +206,9 @@ export default function ActivityTable({
       label: t('Created'),
       onClick: () => {
         setActiveChild('Created');
-        setInLocalStorage('activity', { activity: 'Created' });
+        setInLocalStorage(HOMEPAGE_ACTIVITY_FILTER, {
+          activity: SetTabType.CREATED,
```

Comment:
Do we need to save an object to the local storage? Can't we just `setInLocalStorage(HOMEPAGE_ACTIVITY_FILTER, SetTabType.CREATED)`? Same applies to other uses